Implementation for Issue #841

zakarh opened the pull request:
Hi again, in Issue #845 you said you wanted colors instead of numbers to represent opacity state. I made the changes along with a [video demo](https://www.dropbox.com/s/lf7shsw70knzlnx/Piskel%20%23%20Issue%20841%20v2.mp4?dl=0). This required only a minor change to **addLayerItem** in **LayerListController.js**. Once the layer item is created the style for the alpha icon color is added based on the current opacity for that layer. 

I used the following colors for opacity which match the color style of Piskel.
- Gold (#ffd700) for `α = 1` .
- White (#ffffff) for `0 < α < 1`.
- Grey (#969696) for `α = 0`.

juliandescottes commented on src/js/controller/LayersListController.js:
```diff
@@ -130,6 +130,14 @@
     }
+    var opacity = layer.getOpacity();
+    if (opacity == 1) {
+      layerItem.getElementsByClassName('layer-item-opacity')[0].style.color = '#ffd700';
```
In the codebase we usually use querySelector to find elements, could we replace this with:

```layerItem.querySelector('.layer-item-opacity')```

zakarh commented:
Alright, I modified the code with the appropriate changes.  Also, I squashed the commits.

![image](https://user-images.githubusercontent.com/13757919/45590527-0f28a900-b908-11e8-8fd0-797513b1c400.png)

blurymind commented:
it would be nice if it was a slider and not an input field

juliandescottes commented:
Thanks, merging!

juliandescottes commented:
> it would be nice if it was a slider and not an input field

Yep, right now it's just a `window.prompt` because it was the cheapest option. If anyone wants to implement a fancier widget, nothing against that :)
